src/index.css

```diff
@@ -406,7 +406,7 @@
   align-items:flex-start;
   border-top:1px solid #fff;
   font-weight:300;
-  color:#000;
+  color:#777;
```

Comment:
This CSS element affects multiple headers, on white and black backgrounds.  Grey can be sen on both.